### PR TITLE
sir DD source code

### PR DESCRIPTION
**Pull request by CryoYohan**
favicon

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -38,14 +38,26 @@
...
-				document.getElementById('my_result').innerHTML = '<img id="imageprev" src="'+data_uri+'"/>';
+				document.getElementById('my_result').innerHTML = '<img id="imageprev" src="'+data_uri+"'/>';
 			} );
 			// Update form fields with values from user inputs
 			document.getElementById('my_idno').value = idno.value;
 			document.getElementById('my_fullname').value = fullname.value;
 			document.getElementById('my_course').value = course.value;
 			document.getElementById('my_level').value = level.value;
 		} 
+
```
Comment (CryoYohan) [inline templates/base.html:49]: Sir DD source code

File: templates/base.html [hunk 2]
```diff
@@ -38,14 +38,26 @@
...
+			var data_uri = document.getElementById('imageprev')
+			var idno = document.getElementById('my_idno').value;
+			var fullname = document.getElementById('my_fullname').value;
+			var course = document.getElementById('my_course').value;
+			var level = document.getElementById('my_level').value;
+			
+			Webcam.upload(data_uri,url,function(code,text)){
+				alert('New User Added')
+			}
+		}
```
Comment (CryoYohan) [inline templates/base.html:60]: sir DD source code

File: upload_save_image/app.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+@app.route("/savestudent",methods=['POST'])
+def savestudent()->None:
+    idno:str = request.form['idno']
+    lastname:str = request.form['lastname']
+    firstname:str = request.form['firstname']
+    course:str = request.form['course']
+    level:str = request.form['level']
+    ok:bool = savestudent('students',idno=idno,lastname=lastname,firstname=firstname,course=course,level=level)
+    return redirect("/")
```
Comment (CryoYohan) [inline upload_save_image/app.py:14]: dd

File: upload_save_image/app.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+
+@app.route("/")
+def index()->None:
+    data:list = getall_records('students')
+    return render_template("index.html",students=data,pagetitle="student registration")
+    
+if __name__=="__main__":
+    app.run(debug=True)
```
Comment (CryoYohan) [inline upload_save_image/app.py:23]: dd

**Comment by Javabutdif** [comment]
Pwede mangayo? HAHAHHA

**Comment by CryoYohan** [comment]
Sayoa HAHAHA, sige gooo

**Comment by CryoYohan** [comment]
> Sayoa HAHAHA, sige gooo

**Pull request merged**